src/TestSuite/Constraint/EventFiredWith.php

```diff
@@ -83,14 +83,14 @@ public function matches($other)
         if (count($events) > 1) {
             throw new PHPUnit_Framework_AssertionFailedError(sprintf('Event "%s" was fired %d times, cannot make data assertion', $other, count($events)));
         }
-
+        /** @var \Cake\Event\Event $event */
```

Comment:
I think what he means is /* to use :) Until we can fix up the sniffer here (I got one working, but someone needs to migrate it to CakePHP)